src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Transform.cs

```diff
@@ -204,7 +204,7 @@ public Hashtable PropagatedNamespaces
                 CanonicalXmlNodeList namespaces = null;
                 if (reference != null)
                     namespaces = reference._namespaces;
-                else if (signedXml._context != null)
+                else if (signedXml?._context != null)
```

Comment:
@anthonylangsworth - I'm looking at the code and possibly this should be Context instead of _context. It looks to me like Context represent the actual context but its value is being cached in _context meaning that it can be uninitialized only because noone ever used it